features/FEATURE_UVISOR/includes/uvisor/api/inc/uvisor_deprecation.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2016, ARM Limited, All Rights Reserved
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef __UVISOR_DEPRECATION_H__
+#define __UVISOR_DEPRECATION_H__
+
+#if defined(UVISOR_PRESENT) && UVISOR_PRESENT == 1
+#warning "---- WARNING: You are using FEATURE_UVISOR which is deprecated!! ----"
```

Comment:
`You are using FEATURE_UVISOR which is deprecated since mbed-os-5.9` might be better - follow what we do for API - since what release it is deprecated.

Lets wait for @AnotherButler review

Reply:
Agree with @0xc0170 Uvisor is NOT yet deprecated. The warning should state that the feature is going to be deprecated in 5.10...

Reply:
Can we hold off on warning messages? We're reviewing all of them. Also saying "it's deprecated' without saying what to use instead leaves the user with nothing. 

@simonfordarm

Reply:
> Can we hold off on warning messages? We're reviewing all of them. Also saying "it's deprecated' without saying what to use instead leaves the user with nothing.

@alzix is trying to resolve this to add this here to the message and to the commit message as well.

> Can we hold off on warning messages?

What do you suggest?